### PR TITLE
Adds new Unique Perks to Human Background with an few tweks

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
Personal PR which requires some balance directions:

TODO in others PR:
Generally more tribal clothing/armaments/tools.
Several new perks to very unused backgrounds, making it somehow viable an unique.
Give the Funny Nitrogen bird two different types of mask, an Kriosan like, and another which does not obscures the beak.

	
<hr>
</details>

## Changelog
:cl:
add:  Three new perks available to baseline humans plus a little points rebalance with more to come! 

/:cl:

### CODE REVIEW

File: code/datums/setup_option/backgrounds/origin_human.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	desc = "A catch all term to describe Humans and Abhumans born within Terran Colonies. Those born here have access to much more prestigious and advanced education facilities, \
 	along with various military pathways to choose from. Colony born humans tend to be widely skilled and physically better off than most as a result of clean living and access to \
 	better form of education than most. You may have been born on a colony world or be the child of those that were, in either case you still reap the benefits of good genetics."
-
+a
```
Comment (Trilbyspaceclone) [inline code/datums/setup_option/backgrounds/origin_human.dm:6]:
```suggestion
```

Comment (Guidesu) [reply code/datums/setup_option/backgrounds/origin_human.dm:6]: Ye i still coding fix soon

File: modular_sojourn/general_things/perks.dm [hunk 2]
```diff
@@ -0,0 +1,96 @@
+
+#define PERK_HUMAN_ARMOUR_BOOST /datum/perk/job/natural_armour
+/datum/perk/job/natural_armour
+	name = "Natural Rough Body"
+	desc = "You've been through EVERYTHING due to your frontier origins. Spider bites, random cuts, and slashes on rusted metal and wood, animal clawing and bitting. As a result, via natural adaptation using some sorta exoskeleton or just more resistant fur/skin/scales due to your ancestors, or simply you are stronger due to your rough body, you resist every type of damage just a little bit better than others due to your origins."
+
+/datum/perk/job/natural_armour/assign(mob/living/carbon/human/H)
+	..()
+	holder.brute_mod_perk -= 0.8
```
Comment (Trilbyspaceclone) [inline modular_sojourn/general_things/perks.dm:9]: 80% reduction in brute is a bit much

File: modular_sojourn/general_things/perks.dm [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	holder.burn_mod_perk -= 0.03
```
Comment (Trilbyspaceclone) [inline modular_sojourn/general_things/perks.dm:10]: 3% reduction in burn is a bit odd, could be bit better like 5 or so to help with numbers

File: modular_sojourn/general_things/perks.dm (hunk 2 above)
Comment (Guidesu) [reply modular_sojourn/general_things/perks.dm:9]: oooooooh I SORRY. i thought the 0.8 was like 0.8% damage reduction

**Comment by Guidesu** [comment]
There we go!

File: modular_sojourn/general_things/perks.dm [hunk 4]
```diff
@@ -6,14 +6,14 @@
...
-	holder.oxy_mod_perk -= 0.6
+	holder.brute_mod_perk -= 0.2
+	holder.burn_mod_perk -= 0.2
+	holder.oxy_mod_perk -= 0.1
 
 /datum/perk/job/natural_armour/remove()
-	holder.brute_mod_perk += 0.8
-	holder.burn_mod_perk += 0.03
-	holder.oxy_mod_perk += 0.6
+	holder.brute_mod_perk -= 0.2
```
Comment (Trilbyspaceclone) [inline modular_sojourn/general_things/perks.dm:14]: These should be +

Comment (Guidesu) [reply modular_sojourn/general_things/perks.dm:14]: DONE

File: modular_sojourn/general_things/perks.dm [hunk 5]
```diff
@@ -0,0 +1,96 @@
+
+#define PERK_HUMAN_ARMOUR_BOOST /datum/perk/job/natural_armour
+/datum/perk/job/natural_armour
```
Comment (Trilbyspaceclone) [inline modular_sojourn/general_things/perks.dm:3]: This ADDS 20% more brute and burn damage (10% o2)
flip the assign to be -=
flip the remove to be +=

Comment (Guidesu) [reply modular_sojourn/general_things/perks.dm:3]: done

**Comment by Guidesu** [comment]
diddly diddly done

**Comment by Kazkin** [comment]
Erm... This doesn't seem that balanced on first inspection. Why change the stats so dramatically?

**Comment by Kazkin** [comment]
I guess what I am asking is, for what purpose was this changed so much?

**Comment by Guidesu** [comment]
> I guess what I am asking is, for what purpose was this changed so much?

I dunno.

**Comment by Kazkin** [comment]
Interesting PR but I honestly do not think this is good for the general balance of the game, an interesting first attempt at least. While I do not wish to discourage your contributions, perhaps something on your list that is less ambitious would be better?

**Pull request closed without merging**